CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # PayPal Android SDK Release Notes
 
+## unreleased
+
+* CardPayments
+  * Add `liabilityShift` property to `CardResult`
+  * Parse `liabilityShift` from deep link return URL in `CardClient#approveOrder()` browser-switched flow
```

Comment:
This feels like an implementation detail - is it different from the line above (from a merchant perspective)?

Reply:
That's true. I'll remove this too.